swagger1/src/main/java/com/cerner/beadledom/swagger/SwaggerUiResource.java

```diff
@@ -66,4 +71,21 @@ public Response getSwaggerUiAsset(@PathParam("asset") String assetPath) {
     InputStream stream = getClass().getResourceAsStream("ui-dist/" + assetPath);
     return Response.ok(stream).build();
   }
+
+  private boolean isRequestSecure(SecurityContext securityContext, HttpHeaders httpHeaders) {
+    return securityContext.isSecure()
+        || secureForwardedHeader(httpHeaders.getRequestHeaders().getFirst("Forwarded"))
+        || secureXForwardedProtoHeader(httpHeaders.getRequestHeaders().getFirst("X-Forwarded-Proto"));
+  }
+
+  private boolean secureForwardedHeader(String forwardedHeader) {
```

Comment:
Might be worth including javadoc to explain what this is doing for better understanding